.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "endOfLine": "crlf",
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
Blocking: `trailingComma` should be `none`. We don't need the comma at the end of the line